test/quantization/core/test_quantized_tensor.py

```diff
@@ -864,6 +864,66 @@ def test_qtensor_resize(self):
             q3 = q1.resize(*sizes2)
             q4 = q1.contiguous().resize(*sizes2)
 
+    @unittest.skipIf(not torch.cuda.is_available() or TEST_WITH_ROCM, 'CUDA is not available')
+    def test_cuda_cpu_qimplementation_consistency(self):
```

Comment:
Is this a general test for testing quantize_per_tensor/dequantize on cpu/cuda? It doesn't seem related to the functions added above, unless I'm missing something